ui/components/app/step-progress-bar/step-progress-bar.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import classnames from 'classnames';
+import PropTypes from 'prop-types';
+import { useI18nContext } from '../../../hooks/useI18nContext';
+
+const stages = {
+  PASSWORD_CREATE: 1,
+  SEED_PHRASE_VIDEO: 2,
+  SEED_PHRASE_REVIEW: 3,
+  SEED_PHRASE_CONFIRM: 4,
+  ONBOARDING_COMPLETE: 5,
+};
+export default function StepProgressBar({ stage = 'PASSWORD_CREATE' }) {
+  const t = useI18nContext();
+  return (
+    <div className="container">
```

Comment:
Could we perhaps make use of our `<Box/>` component here?

Reply:
Sure can!

Reply:
Done